src/operator/bufferTime.ts

```diff
@@ -120,7 +120,7 @@ class BufferTimeSubscriber<T> extends Subscriber<T> {
               private scheduler: Scheduler) {
     super(destination);
     const context = this.openContext();
-    this.timespanOnly = bufferCreationInterval == null || bufferCreationInterval < 0;
+    this.timespanOnly = bufferCreationInterval === null || bufferCreationInterval < 0;
```

Comment:
wow, it's my bad. apologize. This isn't legit complete fix and code should be updated more.